app/src/main/java/org/greenstand/android/TreeTracker/database/Migrations.kt

```diff
@@ -55,4 +56,10 @@ val MIGRATION_1_2 = object : Migration(1, 2) {
         }
     }
 
+}
+
+val MIGRATION_2_3 = object : Migration(2, 3) {
+    override fun migrate(database: SupportSQLiteDatabase) {
+        database.execSQL("ALTER TABLE ${PlanterInfoEntity.TABLE} ADD COLUMN ${PlanterInfoEntity.RECORD_UUID} TEXT")
```

Comment:
Shouln't we require a non-null value for `record_uuid`?

Reply:
Update